app/src/test/java/com/kickstarter/ui/activities/compose/PreLaunchProjectPageScreenTest.kt

```diff
@@ -0,0 +1,100 @@
+package com.kickstarter.ui.activities.compose
+
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.test.assertIsNotDisplayed
+import androidx.compose.ui.test.assertTextEquals
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.performClick
+import com.apollographql.apollo.api.CustomTypeValue
+import com.kickstarter.KSRobolectricTestCase
+import com.kickstarter.R
+import com.kickstarter.libs.graphql.DateTimeAdapter
+import com.kickstarter.mock.factories.ProjectFactory
+import com.kickstarter.models.Urls
+import org.junit.Test
+
+class PreLaunchProjectPageScreenTest : KSRobolectricTestCase() {
+
+    private val projectImage = composeTestRule.onNodeWithTag("Project Image")
```

Comment:
hey @hadia question here, what do you thing about having a companion object or some kind of structure on the on `PrelaunchProjectScreen` holding all the tags?

Reply:
Added enum class